resources/helm/overlays/istio-telemetry/kiali/templates/kiali-cr.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     strategy: "openshift"
 
   deployment:
+    accessible_namespaces: []
```

Comment:
Does this need to be initialized to `{{ .Release.Namespace}}`?

Reply:
No. Kiali adds the mesh namespace automatically.